src/examples/colorpickers/ColorpickerDialogFormInput.tsx

```diff
@@ -0,0 +1,91 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { parseToRgb, toColorString } from 'polished';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { ColorpickerDialog, IColor } from '@zendeskgarden/react-colorpickers';
+import { Field, Label, Input, InputGroup } from '@zendeskgarden/react-forms';
+
+const validHex = /^#(?:(?:[0-9A-F]{6}(?:[0-9A-F]{2})?)|(?:[0-9A-F]{3})(?:[0-9A-F]?))$/iu;
+
+const StyledField = styled(Field)`
+  display: inline-block;
+  text-align: ${props => (props.theme.rtl ? 'right' : 'left')};
+`;
+
+const StyledInput = styled(Input)`
+  max-width: ${props => props.theme.space.base * 26}px;
+`;
+
+const StyledInputGroup = styled(InputGroup)`
+  display: flex;
+`;
+
+const toHex = (selectedColor: IColor) => {
+  let colorHex = selectedColor.hex;
+
+  if (selectedColor.alpha === 100) {
+    return colorHex;
+  }
+
+  let alphaHex = Math.round((selectedColor.alpha / 100) * 255).toString(16);
+
+  if (alphaHex.length === 1) {
+    alphaHex = `0${alphaHex}`;
+  }
+
+  if (colorHex.length === 4) {
+    if (alphaHex.charAt(0) === alphaHex.charAt(1)) {
+      alphaHex = alphaHex.charAt(0);
+    } else {
+      colorHex = `#${colorHex.charAt(1)}${colorHex.charAt(1)}${colorHex.charAt(2)}${colorHex.charAt(
+        2
+      )}${colorHex.charAt(3)}${colorHex.charAt(3)}`;
+    }
+  }
+
+  return `${colorHex}${alphaHex}`;
+};
+
+const Example = () => {
+  const [input, setInput] = useState('#1f73b799');
+  const [color, setColor] = useState<string | IColor>('rgba(31, 115, 183, .6)');
+
+  return (
+    <Row justifyContent="center">
+      <Col size="auto">
```

Comment:
Much of the follow-on `StyledXxx` customization for positioning can be alleviated by replacing this with a `StyledCol` that sets `flex-grow: 0;`.